Riot/Modules/MatrixKit/Utils/EventFormatter/MXKEventFormatter.m

```diff
@@ -1053,8 +1053,13 @@ - (NSAttributedString *)attributedStringFromEvent:(MXEvent*)event
                     else if ([event.decryptionError.domain isEqualToString:MXDecryptingErrorDomain]
                         && event.decryptionError.code == MXDecryptingErrorUnknownInboundSessionIdCode)
                     {
-                        // Make the unknown inbound session id error description more user friendly
-                        errorDescription = [VectorL10n noticeCryptoErrorUnknownInboundSessionId];
+                        // Hide the decryption error for event related to another one (like voicebroadcast chunks)
+                        if ([event.relatesTo.relationType isEqualToString:MXEventRelationTypeReference]) {
```

Comment:
How complex it would be here to check whether the "related to" event is actually a state event with `io.element.voice_broadcast_info` type?

In order to hide only the decryption error for the VB chunks

Reply:
@giomfo, I've added the requested change in the last push.